src/main.cpp

```diff
@@ -1,56 +1,51 @@
-#include <stdio.h>
-#include <iostream>
-#include <string.h>
-#include <functional>
-#include <emscripten.h>
 #include <EGL/egl.h>
-#include "SDL.h"
-#include <SDL_image.h>
 #include <GLES2/gl2.h>
-#include "emscripten.h"
-
-GLuint LoadShader ( GLenum type, char * shaderSrc )
-{
-   GLuint shader;
-   GLint compiled;
-   
-   // Create the shader object
-   shader = glCreateShader ( type );
-
-   if ( shader == 0 )
-   	return 0;
-
-   // Load the shader source
-   glShaderSource ( shader, 1, &shaderSrc, NULL );
-   
-   // Compile the shader
-   glCompileShader ( shader );
-
-   // Check the compile status
-   glGetShaderiv ( shader, GL_COMPILE_STATUS, &compiled );
-
-   if ( !compiled ) 
-   {
-      GLint infoLen = 0;
-
-      glGetShaderiv ( shader, GL_INFO_LOG_LENGTH, &infoLen );
-      
-      if ( infoLen > 1 )
-      {
-         char* infoLog = (char *)malloc (sizeof(char) * infoLen );
-
-         glGetShaderInfoLog ( shader, infoLen, NULL, infoLog );
-         std::cout << "Error compiling shader:" << infoLog << std::endl;            
-         
-         free ( infoLog );
-      }
-
-      glDeleteShader ( shader );
-      return 0;
-   }
-
-   return shader;
+#include <SDL2/SDL.h>
```

Comment:
Is SDL still being used I wonder?

Reply:
this is changed by clang format，there is no logic difference，when include SDL.h, giving the namespace is better choice, Because of, either SDL/SDL.h or SDL2/SDK.h should included is implicit

Reply:
@shulong-apple  how about `#include <EGL/egl.h>` ? shouldn't that replace `SDL`? (etiher SDL or SDL2). 

Do we use SDL at all? Did we replace all the APIs to EGL?